pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTestBase.java

```diff
@@ -123,7 +125,7 @@ protected void setUpBase(int numBroker,int numPartitionsOfTC, String topic, int
         admin.tenants().createTenant(NamespaceName.SYSTEM_NAMESPACE.getTenant(),
                 new TenantInfoImpl(Sets.newHashSet("appid1"), Sets.newHashSet(CLUSTER_NAME)));
         admin.namespaces().createNamespace(NamespaceName.SYSTEM_NAMESPACE.toString());
-        admin.topics().createPartitionedTopic(TopicName.TRANSACTION_COORDINATOR_ASSIGN.toString(), numPartitionsOfTC);
+        createTransactionCoordinatorAssign(numPartitionsOfTC);
```

Comment:
I have a question why don't use `admin.topics().createPartitionedTopic(TopicName.TRANSACTION_COORDINATOR_ASSIGN.toString(), numPartitionsOfTC);` to create topic.

Reply:
Ah, because TRANSACTION_COORDINATOR_ASSIGN is not a topic in fact. If we use the old way to create it like `createPartitionedTopic `,  we have to check the name on the broker side to let it pass(like not to create managed ledger or something).

Reply:
Thanks for your reply.